addons/web_editor/static/tests/test_utils.js

```diff
@@ -291,43 +291,34 @@ function wysiwygData(data) {
  *
  * @param {object} params
  */
-async function createWysiwyg(params) {
+async function createWysiwyg() {
     patch();
-    params.data = wysiwygData(params.data);
 
-    var parent = new Widget();
-    await testUtils.mock.addMockEnvironment(parent, params);
+    var params = {data: wysiwygData({})};
+    await testUtils.mock.addMockEnvironment(this, params);
```

Comment:
I don't remember exactly, but changes were necessary to make the JW editor work in the QUnits tests.

Reply:
`this` instead of `new Widget()` is indeed not necessary.

Good catch.

----
`this` is the instance of the fieldHtml widget that will host the wysiwyg.